test/fixtures/EN10168/v0.5.0/files/JSON Attachement Example.base64.txt

```diff
@@ -0,0 +1 @@
+eyJtZXNzYWdlIjoiVGhpcyBpcyBhbiBleGFtcGxlIEpTT04ifQ==
```

Comment:
It would be great if we could keep the filename convention of lowercase with words separated by _ instead of spaces. This would apply to all files in the `fixtures` folder.